admin/class-create-block-theme-admin.php

```diff
@@ -852,4 +909,15 @@ function admin_notice_blank_success() {
 		<?php
 	}
 
+	function admin_notice_variation_success() {
+		$theme_name = $_GET['theme']['name'];
+		$variation_name = get_stylesheet_directory() . '/styles/' . $_GET['theme']['variation'] .'.json';
+
+		?>
+			<div class="notice notice-success is-dismissible">
+				<p><?php printf( esc_html__( 'Your variation of %1$s has been created succesfully. The new variation file is in %2$s', 'create-block-theme' ), esc_html( $theme_name ) , esc_html( $variation_name )  ); ?></p>
```

Comment:
Message is missing theme name for some reason. `Your variation of has been created succesfully.`
There is a small typo `successfully`

Reply:
Thanks for catching this!